actions/locations.js

```diff
@@ -100,7 +101,11 @@ export function createLocation(options) {
   } = options;
 
   return async (dispatch, getState) => {
-    const { authentication: { regionKey: hasRegion }, connected } = getState();
+    const { authentication: { regionKey: hasRegion }, connected, onboarding: { hasAddedLocation } } = getState();
+
+    if (!hasAddedLocation) {
```

Comment:
Should we also dispatch this if they refetch a local location?

Reply:
I don't think so because for onboarding purposes we want this to be set when they create a location while going through the onboarding steps. So say if they had created some locations at some point but then wanted to do the onboarding again (maybe someone new is using their phone) then it won't skip the adding locations step.